setup.py

```diff
@@ -56,6 +56,26 @@
 }
 extras_require['complete'] = sorted(set(sum(extras_require.values(), [])))
 
+
+def _is_test_pypi():
+    """
+    Determine if the Travis CI environment has TESTPYPI_UPLOAD defined and
+    set to true (c.f. .travis.yml)
+
+    The use_scm_version kwarg accepts a callable for the local_scheme
+    configuration parameter with argument "version". This can be replaced
+    with a lambda as the desired version structure is {next_version}.dev{distance}
+    c.f. https://github.com/pypa/setuptools_scm/#importing-in-setuppy
+    """
+    from os import getenv
+
+    return (
+        {'local_scheme': lambda version: ''}
+        if getenv('TESTPYPI_UPLOAD') == 'true'
+        else False
```

Comment:
This will cause a fallback to the version hardcoded in python code on line 81.

Reply:
Yes, this is the intended behavior. We only want scm versioning for TestPyPI.

Reply:
Ah, I see :)
It's an interesting approach which I haven't met before...

Reply:
But be aware that this may cause version mismatches between tag and dist because of human errors

Reply:
Yeah, the main reason is that we currently control everything through [bumpversion](https://github.com/peritus/bumpversion), and I wanted to keep this PR as atomic as possible. We could consider changing things in the future.

Reply:
I'm currently working on a GitHub Action / GitHub App adding a Deploy button to Checks pages of each commit. Such an approach would allow triggering the whole flow with an actual click on GitHub UI ;)

Reply:
We don't have GitHub actions on pyhf yet! But we are very much looking forward to using it when we get it, so we'll be following up with you! :)

Reply:
You need to sign up first https://github.com/features/actions/signup

Reply:
Indeed. I have it for my personal account, but our org doesn't yet.

Reply:
Oh, in this case, I invite you to complete this tutorial https://tutorial.octomachinery.dev :)

Reply:
Thanks! That looks great. :)